### DISCUSSION TITLE
Snap Lens Tool - Need Tutorial

### DISCUSSION THREAD

**Discussion by KungFooJay** [ptrumpis/snap-camera-server#125] [Q&A] [answered]
Hello im so confused on how to download and run snap tool i downloaded python but i dont know how to get it to work can someone teach me how to do this i was able to get everything else downloaded but this is hard. Thank you!

**Comment by Jbla0712** [comment, marked as answer]
⚡If you already have Python installed, you will need to check that you have added it to your PATH. ⚡
Otherwise delete it, it's maybe simpler,  and instal python 3.10. I succeeded on python 3.10 only

https://youtu.be/_f9E1WfVL0U?si=LLrFHj50DrQtfX0O

look in the video description for command or link 


Once installed, you must launch the tool FROM its FOLDER via the command prompt with:
python snap_lens_tool.py 

👉 You can automate the launch with a launcher.bat from tool folder's , ask GPT to help you do this

(It's a simple tutorial without a virtual environment, i prefer conda)
